forms-flow-admin/src/components/roles/roles.tsx

```diff
@@ -358,14 +359,15 @@ const Roles = React.memo((props: any) => {
         </Modal.Body>
         <Modal.Footer>
           <button type="button"
-            className="btn btn-link text-dark" onClick={handleCloseEditRoleModal}>
+            className="btn btn-link text-dark" onClick={handleCloseEditRoleModal} data-testid="create-role-modal-edit-cancel-btn">
```

Comment:
create-role-modal-edit-cancel-btn this is for create or edit. little bit confusing

Reply:
changed